app/assets/stylesheets/application.css.scss

```diff
@@ -368,4 +368,14 @@ a {
   p {
     margin: 0 0 1em;
   }
+
+  .application-mail {
```

Comment:
I tried to center the applicants-e-mail but the position is not quite right. I tried to use the flex syntax: align-items: center; / justify-content: center; (and space-between) but nothing changes.